content/docs/sidebar.json

```diff
@@ -552,46 +552,27 @@
     ]
   },
   {
-    "label": "Studio",
     "slug": "studio",
-    "source": "studio/index.md",
     "icon": "studio",
     "children": [
-      {
-        "label": "How It Works",
-        "slug": "overview"
-      },
+      "how-it-works",
```

Comment:
- Simplifications and consistency changes to sidebar.json

- Several files were renamed to match these ☝🏼